app/models/question_answer.rb

```diff
@@ -9,10 +9,10 @@ class QuestionAnswer < ApplicationRecord
   before_destroy :destroy_redcap_details
 
   def upload_redcap_details
-    UploadRedcapDetailsJob.perform_now(id)
```

Comment:
My understanding is that the difference in these 2 methods (with/without `_now`) has something to do with serialisation of the record. What has exactly changed?